bed/methods_test.go

```diff
@@ -0,0 +1,72 @@
+package bed
+
+import (
+	"testing"
+)
+
+var BedStructTests = []struct {
+	bed        Bed
+	chrom      string
+	chromStart int
+	chromEnd   int
+}{
+	{bed: Bed{Chrom: "chr1", ChromStart: 10, ChromEnd: 20, Name: "", Score: 0, Strand: Positive}, chrom: "chr1", chromStart: 10, chromEnd: 20},
+	{bed: Bed{Chrom: "chr10", ChromStart: 11, ChromEnd: 18, Name: "", Score: 0, Strand: Positive}, chrom: "chr10", chromStart: 11, chromEnd: 18},
+	{bed: Bed{Chrom: "chr12", ChromStart: 12, ChromEnd: 16, Name: "", Score: 0, Strand: Positive}, chrom: "chr12", chromStart: 12, chromEnd: 16},
+}
+
+func TestBed_GetChrom(t *testing.T) {
+	for _, v := range BedStructTests {
+		if v.chrom != v.bed.GetChrom() {
+			t.Errorf("Error in bed methods.go GetChrom(); Expected chrom: %v. Actual chrom: %v.", v.chrom, v.bed.GetChrom())
+		}
+	}
+}
+
+func TestBed_GetChromStart(t *testing.T) {
+	for _, v := range BedStructTests {
+		if v.chromStart != v.bed.GetChromStart() {
+			t.Errorf("Error in bed methods.go GetChromStart(); Expected chromStart: %v. Actual chromStart: %v.", v.chromStart, v.bed.GetChromStart())
+		}
+	}
+}
+
+func TestBed_GetChromEnd(t *testing.T) {
+	for _, v := range BedStructTests {
+		if v.chromEnd != v.bed.GetChromEnd() {
+			t.Errorf("Error in bed methods.go GetChromEnd(); Expected chromEnd: %v. Actual chromEnd: %v.", v.chromEnd, v.bed.GetChromEnd())
+		}
+	}
+}
+
+func TestBed_UpdateLift(t *testing.T) {
+	for _, v := range BedStructTests {
+		v.bed.UpdateLift("chr4", 3, 4)
```

Comment:
Maybe you need to create another variable v_updated, and assign v_updated = v.bed.UpdateLift("chr4", 3, 4)?
I found this page that's Python but kind of relevant: https://stackoverflow.com/questions/53249829/python-keep-changes-on-a-variable-made-within-a-function
I was wondering how permanent the function output is. Maybe when you did v.bed.UpdateLift it worked but just didn't save